### PR TITLE
auto case OCP-32197

### PR DESCRIPTION
**Pull request by QiaolingTang**
Customer scenario for https://bugzilla.redhat.com/show_bug.cgi?id=1796133
case: https://url.corp.redhat.com/8c7ba1f
Log: https://url.corp.redhat.com/6dd12f0

@anpingli PTAL, thanks.

### CODE REVIEW

**Review by anpingli — COMMENTED** [review]
/lgtm

File: features/logging/collector.feature [hunk 1]
```diff
@@ -211,3 +211,41 @@ Feature: collector related tests
...
+    Then the step should succeed
+    And the output should not contain:
+      | Fluentd logs have been redirected to: /var/log/fluentd/fluentd.log |
+      | If you want to print out the logs, use command:                    |
+      | oc exec <pod_name> /usr/local/logs                                 |
+    When I run the :exec client command with:
+      | pod              | <%= cluster_logging('instance').fluentd_ready_pods.first.name %> |
+      | oc_opts_end      |                                                                  |
+      | exec_command     | ls                                                               |
+      | exec_command_arg |  /var/log/                                                       |
```
Comment (liangxia) [inline features/logging/collector.feature:243]: Perhaps
```
  When I execute on the "<%= cluster_logging('instance').fluentd_ready_pods.first.name %>" pod:
    | ls | /var/log/ |
```
Of course, the current one also works.

Comment (QiaolingTang) [reply features/logging/collector.feature:243]: I had hit some error when there didn't have `oc_opts_end`:
```
STDERR:
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl kubectl exec [POD] -- [COMMAND] instead.
```

**Comment by QiaolingTang** [comment]
/hold
Due to https://bugzilla.redhat.com/show_bug.cgi?id=1874746, the `cluster_logging('instance').fluentd_ready_pods.first.name` will return `nil`, the case will always fail.

**Pull request merged**